app/views/ops/_settings_advanced_tab.html.haml

```diff
@@ -7,6 +7,11 @@
         %span.pficon.pficon-error-exclamation
       %strong
         = advanced_tab_warning
+    .alert.alert-info
+      %span.pficon-layered
```

Comment:
You can drop this line, this should be just

```
.alert.alert-info
  %span.pficon.pficon-info
```

(layered means multiple icons over each other)

Hopefully that should fix the icon positioning too..